app/Http/Controllers/EmployeeController.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class EmployeeController extends Controller
+{
+    //
```

Comment:
Make this controller a resource controller

Reply:
Ok